migrations/1688126055_closedJobs.ts

```diff
@@ -0,0 +1,61 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+
+  console.log("Creating new fields/fieldsets");
+
+  console.log(
+    'Create Boolean field "Is closed?" (`closed`) in model "Job" (`job`)'
+  );
+  newFields["10413779"] = await client.fields.create("38231", {
+    label: "Is closed?",
+    field_type: "boolean",
+    api_key: "closed",
+    hint: "Closed jobs will remain online for SEO optimization reasons but will not be directly listed on the website. A banner explaining the position is closed will appear on the top of the page.",
```

Comment:
```suggestion
    hint: "Closed jobs will remain online for search engine optimization reasons but will not be directly listed on the website. A banner explaining the position is closed will appear on the top of the page.",
```
Nitpick

Reply:
It was giving "chai tea"

Reply:
https://media.tenor.com/Pvb52tjijZMAAAAC/spider-man-miles-morales.gif